src/ahbicht/expressions/expression_resolver.py

```diff
@@ -0,0 +1,46 @@
+"""
+This module makes it possible to parse expressions with all its subexpressions
+for example ahb_expressions with condition_expressions or condition_expressions with packages.
+"""
+
+from lark import Transformer, Tree
+
+from ahbicht.expressions.ahb_expression_parser import parse_ahb_expression_to_single_requirement_indicator_expressions
+from ahbicht.expressions.condition_expression_parser import parse_condition_expression_to_tree
+
+
+def expression_resolver(expression: str, resolve_packages: bool = False) -> Tree:
+    """
+    Parses expressions and resolves its subexpressions,
```

Comment:
```suggestion
    Parses any kind of expressions and resolves its subexpressions,
```
damit klar ist, dass hier ahb und condition expression gleichermaßen rein kommen. 

_Edit:_ ok, es kommt eine zeile später :D dann passt es auch für mich